src/renderer/mod.rs

```diff
@@ -62,15 +62,28 @@ impl Renderer {
 
     /// Default terminal styling
     pub const fn styled() -> Self {
+        const BRIGHT_BLUE: Style = if cfg!(windows) {
+            AnsiColor::BrightCyan.on_default().effects(Effects::BOLD)
+        } else {
+            AnsiColor::BrightBlue.on_default().effects(Effects::BOLD)
+        };
         Self {
             stylesheet: Stylesheet {
                 error: AnsiColor::BrightRed.on_default().effects(Effects::BOLD),
-                warning: AnsiColor::BrightYellow.on_default().effects(Effects::BOLD),
-                info: AnsiColor::BrightBlue.on_default().effects(Effects::BOLD),
-                note: Style::new().effects(Effects::BOLD),
+                warning: if cfg!(windows) {
+                    AnsiColor::BrightYellow.on_default().effects(Effects::BOLD)
+                } else {
+                    AnsiColor::Yellow.on_default().effects(Effects::BOLD)
+                },
+                info: BRIGHT_BLUE,
```

Comment:
What does `info` map to in `rustc`?  No level uses `BRIGHT_BLUE`.  Is it a Secondary?

Reply:
It should map to `UnderlineSecondary` or `LabelSecondary`, but `annotate-snippets` doesn't have the concept of primary/secondary, so we print `info: ` before it currently.